packages/forms/src/fields/BooleanField.js

```diff
@@ -39,10 +40,16 @@ function BooleanField(props) {
 	const onChangeHandler = () => {
 		onChange(!formData, uiOptions);
 	};
+	const onBlurHandler = () => {
+		if (onBlur) {
+			onBlur(idSchema && idSchema.$id, formData);
```

Comment:
https://github.com/mozilla-services/react-jsonschema-form/blob/18ac59429e37859478e197039b8502fd07c8a556/src/components/widgets/BaseInput.js

indeed, It's weird but it is the defined format by react-jsonschema-form